### PR TITLE
Rename AccountAddress to Bip44Address

### PR DESCRIPTION
**Pull request by thibault-martinez**

### CODE REVIEW

File: sdk/src/wallet/account/operations/transaction/options.rs [hunk 1]
```diff
@@ -39,7 +39,7 @@ pub enum RemainderValueStrategy {
     /// Move the remainder value to a change address.
     ChangeAddress,
     /// Move the remainder value to any specified address.
-    CustomAddress(AccountAddress),
+    CustomAddress(Bip44Address),
```
Comment (thibault-martinez) [inline sdk/src/wallet/account/operations/transaction/options.rs]: This doesn't seem to actually need a Bip44 one, I could replace with simply Address

Comment (DaughterOfMars) [reply sdk/src/wallet/account/operations/transaction/options.rs]: do it 🗡️

Comment (Thoralf-M) [reply sdk/src/wallet/account/operations/transaction/options.rs]: I think it was initially required to provide it to the ledger nano here https://github.com/iotaledger/iota-sdk/blob/4bf72ecb13c3540895d03cc62eb527241e41675a/sdk/src/client/secret/ledger_nano.rs#L297-L308
But looks like we don't pass this information down anymore
In https://github.com/iotaledger/iota-sdk/blob/4bf72ecb13c3540895d03cc62eb527241e41675a/sdk/src/client/api/block_builder/input_selection/core/remainder.rs#L26 it just gets set to None

**Review by DaughterOfMars — DISMISSED** [review]

File: sdk/src/wallet/account/operations/transaction/options.rs (hunk 1 above)
Comment (thibault-martinez) [reply sdk/src/wallet/account/operations/transaction/options.rs]: Changed

**Review by DaughterOfMars — APPROVED** [review]

**Review by Thoralf-M — APPROVED** [review]

**Review by kwek20 — APPROVED** [review]

**Review by abdulmth — APPROVED** [review]

**Pull request merged**